tests/acceptance/bootstrap/OcmContext.php

```diff
@@ -52,6 +53,18 @@ public function before(BeforeScenarioScope $scope): void {
 		$environment = $scope->getEnvironment();
 		// Get all the contexts you need in this context from here
 		$this->featureContext = $environment->getContext('FeatureContext');
+		$this->notificationContext = $environment->getContext('NotificationContext');
+	}
+
+	/**
+	 * Delete all the inbucket emails
+	 *
+	 * @AfterScenario @email
+	 *
+	 * @return void
+	 */
+	public function clearInbucketMessages():void {
+		$this->notificationContext->clearInbucketMessages();
 	}
```

Comment:
there's error in this method
https://drone.owncloud.com/owncloud/ocis/38923/40/8
```
  │
  ╳  Fatal error: Typed property OcmContext::$notificationContext must not be accessed before initialization (Behat\Testwork\Call\Exception\FatalThrowableError)
  │
  └─ @AfterScenario @email # OcmContext::clearInbucketMessages()
```